protocol/app/params/encoding.go

```diff
@@ -0,0 +1,32 @@
+package params
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/tx"
+)
+
+// EncodingConfig specifies the concrete encoding types to use for a given app.
+// This is provided for compatibility between protobuf and amino implementations.
+type EncodingConfig struct {
```

Comment:
copy of [this](https://github.com/cosmos/cosmos-sdk/blob/20856950784e7875f0b9b806c97c793f6dd5e469/simapp/params/encoding.go#L11) and replaces `"cosmossdk.io/simapp/params"`